### PR TITLE
Use @metamask/controllers

### PR DESCRIPTION
**Pull request by rickycodes**
remove `gaba` and use `@metamask/controllers`

### CODE REVIEW

File: app/components/UI/Transactions/index.js [hunk 1]
```diff
@@ -18,11 +18,11 @@ import TransactionElement from '../TransactionElement';
 import Engine from '../../../core/Engine';
 import { showAlert } from '../../../actions/alert';
 import TransactionsNotificationManager from '../../../core/TransactionsNotificationManager';
-import { CANCEL_RATE, SPEED_UP_RATE } from 'gaba';
+import { CANCEL_RATE, SPEED_UP_RATE } from '@metamask/controllers';
 import { renderFromWei } from '../../../util/number';
 import { safeToChecksumAddress } from '../../../util/address';
 import Device from '../../../util/Device';
-import { hexToBN } from 'gaba/dist/util';
+import { hexToBN } from '@metamask/controllers/dist/util';
```
Comment (whymarrh) [inline app/components/UI/Transactions/index.js:25]: Can we not dig into the directory structure of the package?

There is a named `util` export: https://github.com/MetaMask/controllers/blob/v2.0.0/src/index.ts#L2

File: app/components/UI/TxNotification/index.js [hunk 2]
```diff
@@ -12,13 +12,13 @@ import Device from '../../../util/Device';
...
 import { strings } from '../../../../locales/i18n';
-import { CANCEL_RATE, SPEED_UP_RATE, BN } from 'gaba';
+import { CANCEL_RATE, SPEED_UP_RATE, BN } from '@metamask/controllers';
 import ActionContent from '../ActionModal/ActionContent';
 import TransactionActionContent from '../TransactionActionModal/TransactionActionContent';
 import { renderFromWei } from '../../../util/number';
 import Engine from '../../../core/Engine';
 import { safeToChecksumAddress } from '../../../util/address';
-import { hexToBN } from 'gaba/dist/util';
+import { hexToBN } from '@metamask/controllers/dist/util';
```
Comment (whymarrh) [inline app/components/UI/TxNotification/index.js:21]: Can we not dig into the directory structure of the package?

There is a named `util` export: https://github.com/MetaMask/controllers/blob/v2.0.0/src/index.ts#L2

**Review by whymarrh — CHANGES_REQUESTED** [review]

**Comment by rickycodes** [comment]
@whymarrh this should be good now

**Review by whymarrh — APPROVED** [review]
LGTM!

**Pull request merged**